client/src/app/site/pages/meetings/services/active-meeting.subscription.ts

```diff
@@ -1,35 +1,52 @@
+import 'src/app/site/services/model-request-builder';
```

Comment:
Without this import I get an error in BaseComponent on runtime. I do not know why and did not put too much effort into it. We might want to open an issue for that.